backend/src/utils/resourceUtils.ts

```diff
@@ -24,45 +24,65 @@ import {
   ResourceWatcherTimeUpdate,
 } from './resourceWatcher';
 import { getComponentFeatureFlags } from './features';
-import { yamlRegExp } from './constants';
+import { yamlRegExp, blankDashboardCR } from './constants';
 import { getIsAppEnabled, getRouteForClusterId } from './componentUtils';
+import fastify from 'fastify';
+
+const _ = require('lodash');
 
 const dashboardConfigMapName = 'odh-dashboard-config';
 const consoleLinksGroup = 'console.openshift.io';
 const consoleLinksVersion = 'v1';
 const consoleLinksPlural = 'consolelinks';
 const enabledAppsConfigMapName = process.env.ENABLED_APPS_CM;
 
-let dashboardConfigWatcher: ResourceWatcher<V1ConfigMap>;
+let dashboardConfigWatcher: ResourceWatcher<DashboardConfig>;
 let subscriptionWatcher: ResourceWatcher<SubscriptionKind>;
 let appWatcher: ResourceWatcher<OdhApplication>;
 let docWatcher: ResourceWatcher<OdhDocument>;
 let kfDefWatcher: ResourceWatcher<KfDefApplication>;
 let buildsWatcher: ResourceWatcher<BuildStatus>;
 let consoleLinksWatcher: ResourceWatcher<ConsoleLinkKind>;
 
-const DEFAULT_DASHBOARD_CONFIG: V1ConfigMap = {
-  metadata: {
-    name: dashboardConfigMapName,
-    namespace: 'default',
-  },
-  data: {
-    enablement: 'true',
-    disableInfo: 'false',
-    disableSupport: 'false',
-    disableClusterManager: 'true',
-    disableTracking: 'true',
-    disableBYONImageStream: 'true',
-    disableISVBadges: 'true',
-    disableAppLauncher: 'true',
-  },
+const fetchDashboardCR = (fastify: KubeFastifyInstance): Promise<DashboardConfig[]> => {
+  const crResponse: Promise<DashboardConfig[]> = fastify.kube.customObjectsApi
+    .listNamespacedCustomObject(
```

Comment:
the name is hardcoded elsewhere, so we should be consistent and get by name.  Preferably by env var and default to the hardcoded version you are using elsewhere.  As it is now, if there happen to be multiple resources, you're just going to end up using the first one Kube returns.